client/my-sites/plans/jetpack-plans/product-lightbox/product-details.tsx

```diff
@@ -26,31 +29,43 @@ const ProductDetails: React.FC< ProductDetailsProps > = ( { product } ) => {
 		setContentStyle( { maxHeight: `${ height }px` } );
 	}, [ setContentStyle ] );
 
+	// Should be removed once translations are ready.
```

Comment:
You can add the label `String Freeze` to your PR to ensure new strings are translated before merging.

Reply:
Since this is an urgent update and we needed to merge this soon while translations are being work on, i18n team [suggested this temporary checking](https://github.com/Automattic/wp-calypso/pull/70585#issuecomment-1333350882) and limit the changes only for english only.